src/doc_scan_service/session/create/sdk.config.builder.js

```diff
@@ -165,6 +165,61 @@ class SdkConfigBuilder {
     return this;
   }
 
+  /**
+   * Allows configuring the number of attempts permitted for text extraction on an ID document
+   *
+   * @param {string} category the category of retries
+   * @param {number} retries the number of retries (more than 0)
+   *
+   * @returns {this}
+   */
+  withIdDocumentTextExtractionCategoryRetries(category, retries) {
+    Validation.isString(category, 'category');
+    Validation.isInteger(retries, 'retries');
+    Validation.notLessThan(retries, 1, 'retries');
+
+    const attemptsTarget = DocScanConstants.ID_DOCUMENT_TEXT_DATA_EXTRACTION;
```

Comment:
Is there a plan to have different attempts targets in the future?

Reply:
Hum, not that I can see in the documentation we have, but it could happen.
Would that affect the implementation ? I suppose the creation of the overall `attemptsConfiguration` map could then be generalise, so you'd have a getOrCreate() internal method somewhere so it is not repeated in every different targets method.